src/js/modules/comments/components/Comment.js

```diff
@@ -35,8 +37,20 @@ const styles = {
   },
 };
 
-const Comment = ({ classes, comment, users }) => {
-  const user = users[comment.userId];
+/* dateFormat:
+ * longDate: June 9, 2007
+ * shortTime: 5:46 PM
+ */
+
+const Comment = ({ classes, comment }) => {
+  const { user, publishedAt } = comment;
+  // If the timestamp is the same day (< 24 hours difference), use the
+  // shortTime format. If not, use the longDate format.
+  const dateSetting =
```

Comment:
why `dateSetting` and not `dateFormat`

Reply:
dateFormat is a function I imported at the top of the file.